controllers/activemqartemisaddress_controller_deploy_operator_test.go

```diff
@@ -137,7 +138,9 @@ var _ = Describe("Address controller DO", Label("do"), func() {
 					Version: "v1",
 					Kind:    "Pod",
 				}
-				restClient, err := apiutil.RESTClientForGVK(gvk, false, restConfig, serializer.NewCodecFactory(scheme.Scheme))
+				httpClient, err := rest.HTTPClientFor(restConfig)
+				Expect(err).To(BeNil())
+				restClient, err := apiutil.RESTClientForGVK(gvk, false, restConfig, serializer.NewCodecFactory(scheme.Scheme), httpClient)
```

Comment:
Why is this change required?

Reply:
That's also because some of the new dependencies API has changed.